hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/Trash.java

```diff
@@ -69,7 +69,7 @@ public Trash(FileSystem fs, Configuration conf) throws IOException {
    * Hence we get the file system of the fully-qualified resolved-path and
    * then move the path p to the trashbin in that volume,
    * @param fs - the filesystem of path p
-   * @param p - the  path being deleted - to be moved to trasg
```

Comment:
Since we are updating this, would you also please remove the extra space between "the" and "path"?